catholic-gators-dev-firebase-adminsdk-q2yci-0f7d8fa061.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "service_account",
+  "project_id": "catholic-gators-dev",
+  "private_key_id": "0f7d8fa0614ab80c0ef91f6fc06ac7209de3cf42",
+  "private_key": "-----BEGIN PRIVATE KEY-----\nMIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQDFUKHT66TESDUD\nZQKDG4WPfJBp1weGyFL6omRbk8WsHjgD4ke3BKBzTZ9a5eMDjDr8MCTOl+Vqdm+8\nSsq56FhgfP+M1d4EqEcCNLcrA6iNoXL1eFL0/McLffbG+YM94nJocEleZTjmLE0k\nS17zDt7vMGvBR+9SEeJEGMk1XO19XmAmJAWUnUW5CCNcqsf65SMIBLpWS3HCaznF\nHdi2yBHC++1tYJCBv6EmNWmPTHOXt2VZnu8SMQk5TwUPTiYl7D3dJdtORbyQYxll\nS2l86/VFXt+bo+vfrJztc0r6IcB9/oOJ1Obeb7dhAGpt1YaNJV8rt15PcrSJ8RMo\nLtkYD7s5AgMBAAECggEABuwM21GEXJ7wXiu+XnFaGoYrDk18/ftb3t9Q0D4jRIYk\nXq2C6M6kntrppN+U+thB7vw05FsDkTwxsAw7J7G5HPxxniQYF3d8Uz1ccim9EWfp\nL6eemGngRTzQahtRgXum2hdbWZCJQhjqhWQQAwyh4UgEdbnDcXB/bLZmY6T8j2Xu\n9kf/e6AXqvleh32iNynVHdCeuQHIpDtcLsCD3HknlL4VqlVfsvl2eqgFWRuKcLAz\nPiUOJor5R69D0W3Q4J7Pk0KDT88J6fnqSrWhWjcuANaU3UqZvsnWaTX+LZZgu1u0\n5ldwvLzFVFoEa1i7hSvFD7OjGbx4Rn+tVktEgMXz4QKBgQD4U4eE+g0Bww6XVRyV\n/jP4D+Fj+ySfh60gDzqpOuigFpNH1xpL5vpa5GUM9bXNe1epRl9vMJtdZCIQ+a0b\n68jR2FJz2WuwadCcM6b0d4O7XlQKNU1YzEZbEOyC54mmWwUPkQt5GPepxZ6eFk0p\nCpdeViT8V8vxv/cpTqK2alWBoQKBgQDLaY9nVAfqp0MwRH/n76/ev/4y8pUbhY2w\nJC6jWvfWOyMGDNq3pIJjWNlIawCTdmyE9kA0Tr8K1eiERVOCLrRRYpyV8xuT+mL4\nRhjc+v4dwfRVOdvsCk0yK4mlq+X+zXP9Oqb8ba9ikJWkRbNGw6V1Jyq2jKAe3L5R\ny4/N/rICmQKBgQDdEj/LenCp6OTMhND2F2BcroaRY7uu6jKOIACxN+hw3KgxhKW8\nfNzUng/4AabgT7Yy1JuReUxXUBWj1yLtgHM/nb2JtT3Rx+IXrUQsz1lPLSLUE6/Q\nJctctyxXPZmlp9JRkWcNX9TcJzqWsnPu3ab0LOywWdkOVz/KXl8s1m1ZAQKBgH8V\nj1ueG2gWYhrY4RSWIknHp9hKyK+uLAydPmoe3nKA9evEwi0z4aaSDwZ4m56FwVz8\nwLYtcJwjcqaY7YiLB2tFcdEZVuSH7adxm/0L7xHGaJGKmYUKFpq+qrvP/tzHnazb\nJRLWsFmjPWqphU0z1L6x3rS4fR1rCpjt1gV8F9kZAoGBAKWR5Fq1DUih1TQDSYNs\ncEOfORGvJVB2tQI1ZqaWO2P1QlJMjp+35qmYbwkDbHZ8SA8kwN56XC8OAEWCkSOk\npDZ4iLVvsn5Jli0sW+neUmDJiJmmrvZ/2B1TRw2CRRrPq/gswNMcbG5zalJn+RcM\nMBbDx5fgTIIktsH5XU9l3YVP\n-----END PRIVATE KEY-----\n",
```

Comment:
Are we %100 sure this is allowed to be in the frontend? Because this is a key. If so, how do we control the flow of data into and out of our DB?

Reply:
@RyanClementsHax

Reply:
Lol, just got this email: 
![capture](https://user-images.githubusercontent.com/17769391/52833519-0f8dc180-30ab-11e9-9d7b-5b11f994c992.PNG)

Reply:
Yeah we shouldn’t commit any of the backend keys we need to do what that email says